src/ignitus-Shared/ignitus-DesignSystem/shared.ts

```diff
@@ -91,4 +91,27 @@ export const flexibleRowDiv = styled.div`
   justify-content: center;
 `;
 
+export const TextIconContainer = styled.div`
```

Comment:
For Inspiration please have a look here. http://www.ignitus.org/interface/sideNavigation, here icons are used along with the text, with no involvement of position property at all. Here is the code for the same https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-SideNavigation/ignitus-DashboardSideNavigation/Components/index.tsx#L39